specification/purview/data-plane/Azure.Analytics.Purview.DataQuality/preview/2022-09-01-preview/purviewDataQuality.json

```diff
@@ -0,0 +1,3820 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Data Quality",
+    "description": "Purview Data Quality Service is a fully managed cloud service whose users can run Data Quality Job to get some score. This spec defines REST API of Purview Data Quality Service.",
+    "version": "2022-09-01"
+  },
+  "paths": {
+    "/accounts/{accountId}/alerts": {
+      "post": {
+        "tags": [
+          "DataQuality Alerts"
+        ],
+        "description": "Create a DQ Alert.",
+        "operationId": "Alert_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountId",
+            "description": "Get Account Id",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
You should probably create a defined parameter for `accountId` and just reference that from the 41 different places it is used. You should also annotate this parameter with `x-ms-parameter-location: client` so that code generation will make it a parameter on the client constructor and not on every method.